### ISSUE TITLE
error trying to create a k8s_cluster

### ISSUE THREAD

**Issue by mattsawyer77** [volterraedge/terraform-provider-volterra#25]
I'm using the 0.1.0 provider, and I'm attempting to create a `k8s_cluster`, but I'm running into an error:

```
Error: error creating K8SCluster: Creating object: Unsuccessful POST at URL https://<redacted>.console.ves.volterra.io/api/public/namespaces/matt-test/k8s_clusters, status code 400, body {
  "code": 3,
  "message": "Field spec.insecure_registries_choice should be not nil, got nil in request.",
  "details": [
    {
      "@type": "type.googleapis.com/ves.io.stdlib.server.error.Error",
      "code": "UNKNOWN",
      "details": "Field spec.insecure_registries_choice fails rule ves.io.schema.rules.message.required constraint true due to value \u003cnil\u003e",
      "timestamp": "2021-02-19 23:35:37.399449404 +0000 UTC m=+98809.477567430"
    }
  ]
}, err %!s(<nil>)
```

here is the terraform I'm trying to use:

```hcl
resource "volterra_k8s_cluster" "matt_test" {
  name                              = "matt-test"
  namespace                         = volterra_namespace.matt_test.name
  use_default_cluster_role_bindings = true
  use_default_cluster_roles         = true
  use_default_psp                   = true
  no_global_access                  = true
  no_local_access                   = true
}
```

I attempted to add in the property as follows:
```hcl
  insecure_registries_choice        = false
```
but this also doesn't work:

```
Error: Unsupported argument

  on main.tf line 45, in resource "volterra_k8s_cluster" "matt_test":
  45:   insecure_registries_choice        = false

An argument named "insecure_registries_choice" is not expected here.
```

**Comment by pupapaik** [comment]
Thank you for this. This is old schema in client library caused by last week upgrade. We will fix this asap.

**Comment by madhukar32** [comment]
@mattsawyer77: This is fixed with the new version v0.1.1
